src/admin/components/card/performance.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  ResponsiveContainer,
+} from 'recharts';
+
+const Performance = () => {
+  const data = [
```

Comment:
Extract to a separate file

Reply:
created separate file for the dummy performance data.